src/v/cluster/topic_table_probe.cc

```diff
@@ -9,14 +9,72 @@
 
 #include "cluster/topic_table_probe.h"
 
+#include "cluster/logger.h"
 #include "cluster/topic_table.h"
 #include "config/configuration.h"
+#include "config/node_config.h"
 #include "prometheus/prometheus_sanitize.h"
 
 namespace cluster {
 
 topic_table_probe::topic_table_probe(const topic_table& topic_table)
-  : _topic_table(topic_table) {}
+  : _topic_table(topic_table) {
+    setup_metrics();
+}
+
+void topic_table_probe::setup_metrics() {
+    if (
+      config::shard_local_cfg().disable_public_metrics()
+      || ss::this_shard_id() != 0) {
+        return;
+    }
+    namespace sm = ss::metrics;
+    _metrics.add_group(
+      prometheus_sanitize::metrics_name("node:movements"),
```

Comment:
let's keep metric namespaces in line with the code structure, i.e. group name "cluster:partitions", gauge names: "moving_to_node", "moving_from_node"

Reply:
fixed